provider-ci/internal/pkg/templates/bridged-provider/.github/workflows/main.yml

```
@@ -181,6 +181,13 @@ jobs:
       #{{- end }}#
     runs-on: #{{ if .Config.runner.publish }}##{{- .Config.runner.publish }}##{{ else }}##{{- .Config.runner.default }}##{{ end }}#
     steps:
+    - name: Free Disk Space (Ubuntu)
+      uses: jlumbroso/free-disk-space@main
+      with:
+        # this might remove tools that are actually needed,
+        # if set to "true" but frees about 6 GB
```

let's explain why we're setting it to `false` rather than saying what it would do if we were setting it to `true`:

```suggestion
        # we are setting this to false to avoid removing tools that are actually needed,
        # but if set to "true" it could free up about 6 GB
```